packages/linux-drivers/amlogic/fd628-aml/package.mk

```diff
@@ -17,12 +17,12 @@
 ################################################################################
 
 PKG_NAME="fd628-aml"
-PKG_VERSION="f8bd53c"
-PKG_SHA256="cf117c8512d7ecf624e1fd8c4a72264c17927637d8e9b191c0a611163a0a2b8f"
+PKG_VERSION="84788fb"
+PKG_SHA256="197d248e45ed497053c705ed16eff806cc77d48ace8bbffc21d378c1621e23f7"
 PKG_ARCH="arm aarch64"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/tanixbox/tx3mini_linux_fd628"
-PKG_URL="https://github.com/tanixbox/tx3mini_linux_fd628/archive/$PKG_VERSION.tar.gz"
+PKG_URL="https://github.com/arthur-liberman/linux_fd628/archive/$PKG_VERSION.tar.gz"
 PKG_SOURCE_DIR="tx3mini_linux_fd628-$PKG_VERSION*"
```

Comment:
should be `PKG_SOURCE_DIR="linux_fd628-$PKG_VERSION*"` after the package/repo rename